client/blocks/reader-post-card/blocked.jsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
```

Comment:
Could we use [the analytics middleware](https://wpcalypso.wordpress.com/devdocs/client/state/analytics/README.md?term=state%20analytics) here instead? It lets you `recordGoogleEvents()` and `bumpStats()` using Redux